packages/devtools_app/lib/src/network/network_controller.dart

```diff
@@ -153,6 +168,8 @@ class NetworkController {
     // wall-time a request was made. This won't be 100% accurate, but it should
     // easily be within a second.
     _timelineMicrosOffset = DateTime.now().microsecondsSinceEpoch - timestamp;
+    await allowedError(serviceManager.service
+        .setVMTimelineFlags(<String>['GC', 'Dart', 'Embedder']));
```

Comment:
can we remove the type here?

Reply:
done. also removed the type from all other places in devtools we used this api